### PR TITLE
Page/movie details

### PR DESCRIPTION
**Pull request by yousefo1995**

### CODE REVIEW

File: src/pages/MoviePage.js [hunk 1]
```diff
@@ -11,30 +11,73 @@ import StyledCardButton from "../components/MovieCard/StyledCardButton/StyledCar
...
           </Grid>
-          <Grid item container md={12} lg={2} spacing={0.5}>
+          <Grid item container md={12} lg={2.25} spacing={0.5}>
             <Grid item xs={6} lg={12}>
-              <IconCard icon="videos">13 videos</IconCard>
+              <IconCard icon="videos">3 videos</IconCard>
             </Grid>
             <Grid item xs={6} lg={12}>
-              <IconCard icon="photos">13 photos</IconCard>
+              <IconCard icon="photos">22 photos</IconCard>
```
Comment (owaysasi) [inline src/pages/MoviePage.js]: remove the fixed values

File: src/pages/MoviePage.js [hunk 2]
```diff
@@ -11,30 +11,73 @@ import StyledCardButton from "../components/MovieCard/StyledCardButton/StyledCar
...
+                imagePath={data.backdrop_path}
                 showWishlistBtn={false}
               />
             </Box>
           </Grid>
-          <Grid item container md={12} lg={2} spacing={0.5}>
+          <Grid item container md={12} lg={2.25} spacing={0.5}>
             <Grid item xs={6} lg={12}>
-              <IconCard icon="videos">13 videos</IconCard>
+              <IconCard icon="videos">3 videos</IconCard>
```
Comment (owaysasi) [inline src/pages/MoviePage.js]: remove the fixed values

File: src/pages/MoviePage.js [hunk 3]
```diff
@@ -11,30 +11,73 @@ import StyledCardButton from "../components/MovieCard/StyledCardButton/StyledCar
...
+        setData(response.data);
+      })
+      .catch(function (error) {
+        console.error(error);
+      });
+  }, []);
+
   return (
-    <Box className="gradiant">
-      {/*  // xl ? md ? none */}
```
Comment (owaysasi) [inline src/pages/MoviePage.js:17]: remove comments

**Review by owaysasi — CHANGES_REQUESTED** [review]

File: src/pages/MoviePage.js (hunk 1 above)
Comment (yousefo1995) [reply src/pages/MoviePage.js]: done

File: src/pages/MoviePage.js (hunk 2 above)
Comment (yousefo1995) [reply src/pages/MoviePage.js]: done

File: src/pages/MoviePage.js (hunk 3 above)
Comment (yousefo1995) [reply src/pages/MoviePage.js:17]: done

**Pull request merged**